lib/rules/sort-keys.js

```diff
@@ -73,12 +73,15 @@ const isValidOrders = {
 
 module.exports = {
     meta: {
+        type: "style",
```

Comment:
sort-imports is "suggestion" but sort-keys is "style"-- feels inconsistent.

Reply:
Yeah, this seems like it should be a suggestion. Let's see what others think.